README.md

```diff
@@ -56,6 +56,34 @@ Or with Docker exec:
 ```sh
 docker exec -it immich-auto-stack /script/immich_auto_stack.sh
 ```
+
+## Customizing the criteria
+
+Configurable criteria allows for the customization of how files are grouped
+The default is equivalent to:
+
+```python
+lambda x: (
+  x["originalFileName"].split(".")[0],
+  x["localDateTime"]
+)
+```
+
+To override the default, pass a new configuration file into docker via
+The CRITERIA env var.
+
+```shell
+docker -e CRITERIA='[{"key": "originalFileName", "split": {"key": "_", "index": 0}}]' ...
```

Comment:
This must be `"key": "."` instead of `"key": "_"`

Reply:
That was an example of an override to deviate from the default. Perhaps not the best example because it was so close to the default.